src/main/scala/io/iohk/ethereum/blockchain/sync/SyncController.scala

```diff
@@ -65,17 +65,21 @@ class SyncController(
   }
 
   def handlePeerUpdates: Receive = {
-    case PeerManagerActor.PeersResponse(peers) =>
-      peers.foreach(_.ref ! PeerActor.GetStatus)
+    case peers: PeerManagerActor.Peers =>
+      val newHandshakedPeers = peers.handshaked.filterNot(p => handshakedPeers.contains(p._1.ref))
+      newHandshakedPeers.foreach(p => context watch p._1.ref)
```

Comment:
Minor detail: As we later start watching all handshaked peers in `if (!handshakedPeers.contains(peer.ref)) context watch peer.ref`, is watching this peers necessary here?